doc/tech/roadmap.md

```diff
@@ -104,11 +113,20 @@ The CNTT is collaboratively working with other standard bodies within the Open S
 - CNCF
 - ETSI NFV ISG
 
+<a name="6.4.1"></a>
+### 6.4.1 Dependencies with OPNFV and OVP
+
 The OpenStack based work stream in CNTT community works closely with OPNFV by leveraging and evolving OPNFV continuous integration pipeline with automation installer and testing capabilities. The Reference Implementation (RI1) has dependencies with other industry communities like OPNFV as shown [here](../ref_impl/cntt-ri/chapters/chapter01.md#1.3) and OVP as shown [here](../ref_impl/cntt-ri/chapters/chapter01.md#1.3). For information on the relationship to other communities, please read [Reference Implementation Chapter 01](../ref_impl/cntt-ri/chapters/chapter01.md#1.3). 
 
 <p align="center"><img src="./figures/Relation_to_other_communities.png" alt="Relation to other communities" title="Relation to other communities" width="60%"/></p>
 <p align="center"><b>Figure 4:</b> Relation to other communities.</p>
 
+<p align="center"><img src="./figures/tech_roadmap_cad.png" alt="Relation to other communities" title="Relation to other communities" width="100%"/></p>
+<p align="center"><b>Figure 5:</b> CNTT Roadmap relation to OPNFV and OVP.</p>
```

Comment:
Figure 5 -- same as Figure 3 -- defines Baldy and Baraque differently then Figure 2.

Reply:
fixed